### PR TITLE
Add ORT dns local bind inference

### PR DESCRIPTION
**Pull request by rob05c**
Adds a flag to ORT to set the ATS dns local binding to the service address(s).
This is necessary when there are multiple addresses and interfaces on the host, ATS may arbitrarily choose an undesirable address to bind to, unless the config is explicit.

Includes tests.
Includes docs, ort flags are documented in the app itelf.
Includes changelog.

- [x] This PR is not related to any other Issue

## Which Traffic Control components are affected by this PR?
- Traffic Ops ORT

## What is the best way to verify this PR?
Run tests.
Run ORT with the new `dns_local_bind` flag, verify records.config has `dns.local_ipv4` and `dns.local_ipv6` matching the server's Interface Service Addresses.

## If this is a bug fix, what versions of Traffic Control are affected?
Not a bug fix.

## The following criteria are ALL met by this PR
- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

## Additional Information

### CODE REVIEW

File: traffic_ops_ort/atstccfg/config/config.go [hunk 1]
```diff
@@ -104,6 +105,7 @@ func GetCfg() (Cfg, error) {
 	disableProxyPtr := flag.BoolP("traffic-ops-disable-proxy", "p", false, "Whether to not use the Traffic Ops proxy specified in the GLOBAL Parameter tm.rev_proxy.url")
 	dirPtr := flag.StringP("dir", "D", "", "ATS config directory, used for config files without location parameters or with relative paths. May be blank. If blank and any required config file location parameter is missing or relative, will error.")
 	viaReleasePtr := flag.BoolP("via-string-release", "", false, "Whether to use the Release value from the RPM package as a replacement for the ATS version specified in the build that is returned in the Via and Server headers from ATS.")
+	dnsLocalBind := flag.BoolP("dns-local-bind", "", false, "Whether to use the server's Service Addresses to set the ATS DNS local bind address.")
```
Comment (serDrem) [inline traffic_ops_ort/atstccfg/config/config.go:108]: Bucking the trend to not name new new var dnsLocalBindPtr makes it look different from the others, when it's really not.

**Review by serDrem — APPROVED** [review]
did not know adding an option was that complicated.

File: traffic_ops_ort/atstccfg/config/config.go (hunk 1 above)
Comment (rob05c) [reply traffic_ops_ort/atstccfg/config/config.go:108]: Eh, I originally did the `viaReleasePtr` thing. I think that was a mistake now, it adds complexity for no value. You're right, consistency is good, they should really all be changed

Comment (rob05c) [reply traffic_ops_ort/atstccfg/config/config.go:108]: Changed

**Pull request merged**